string mission_id
string uav_id

float64 pickup_lat
float64 pickup_lon
float64 pickup_alt

float64 dropoff_lat
float64 dropoff_lon
float64 dropoff_alt

string priority     # normal / high
int64 deadline_ts   # Unix 时间戳，可为 0 表示无
string note

